pkg/hub/lease/controller.go

```diff
@@ -95,6 +100,10 @@ func (c *leaseController) sync(ctx context.Context, syncCtx factory.SyncContext)
 			return err
 		case err == nil:
 			gracePeriod := time.Duration(leaseDurationTimes*cluster.Spec.LeaseDurationSeconds) * time.Second
+			// FIX: #183 avoid gracePeriod is zero, will non-stop update ManagedClusterLeaseUpdateStopped condition.
```

Comment:
could we comment here like:

```
set default grace period if it is zero in managedcluster
```
And I would also thinks this should be set as default in crd spec.

Reply:
this [pr](https://github.com/open-cluster-management-io/api/pull/111) fix this.